terraform/transform_resource_test.go

```diff
@@ -37,11 +37,33 @@ func TestResourceCountTransformer_countNegative(t *testing.T) {
 	}
 }
 
+func TestResourceCountTransformer_deps(t *testing.T) {
+	cfg := testModule(t, "transform-resource-count-deps").Config()
+	resource := cfg.Resources[0]
+
+	g := Graph{Path: RootModulePath}
+	{
+		tf := &ResourceCountTransformer{Resource: resource}
+		if err := tf.Transform(&g); err != nil {
+			t.Fatalf("err: %s", err)
+		}
+	}
+
+	actual := strings.TrimSpace(g.String())
+	expected := strings.TrimSpace(testResourceCountTransformDepsStr)
+	if actual != expected {
+		t.Fatalf("bad:\n\n%s", actual)
+	}
+}
+
 const testResourceCountTransformStr = `
 aws_instance.foo #0
-  aws_instance.foo #2
 aws_instance.foo #1
-  aws_instance.foo #2
 aws_instance.foo #2
-  aws_instance.foo #2
```

Comment:
Related to the above comment about test cases, I'm not sure how this ever happened.